frontend/src/locales/scripts/get-validated-locales.js

```diff
@@ -13,47 +13,71 @@ const { addFetchedTranslationStatus } = require("./get-translations-status")
  * @returns {{
  * translated: import('./types').I18nLocaleProps[],
  * untranslated: import('./types').I18nLocaleProps[]
+ * invalid: import('./types').I18nLocaleProps[],
  * }}
  */
 const getValidatedLocales = async () => {
   const result = {
     translated: [],
     untranslated: [],
+    invalid: [],
   }
   const updatedLocaleList = await addFetchedTranslationStatus(localesList)
   const allLocales = Object.values(updatedLocaleList).map((locale) => ({
+    /* Nuxt i18n fields */
+
     code: locale.slug,
-    name: locale.name,
-    nativeName: locale.nativeName ?? locale.name,
-    iso: locale.langCodeIso_639_1,
-    wpLocale: locale.wpLocale,
     dir: locale.textDirection || "ltr",
-    translated: locale.translated,
     file: `${locale.slug}.json`,
+    iso: locale.langCodeIso_639_1 ?? undefined,
```

Comment:
Should we throw an error if iso is not available? I think we use it for the translation banner link, and the HTML lang attribute, right?

Reply:
There are many languages that do not have an ISO code. If that is used in some places, falling back to the `code` might be better than raising an error, especially because an ISO standard cannot be changed, not by us.